README.md

```diff
@@ -1,22 +1,68 @@
+| Raid Group | The collective noun for all contributors to the Raid |
+| Raid Repo | The repository for the current Raid |
+
+## Storyline Quest (mission) :scroll:
+
+Objectives:
+
+* To make contributing to OSS enjoyable, engaging, and with a low barrier-to-entry for those who have never done it before
+* To gamify the process of mob programming OSS issues
+* To have a high impact in a small amount of time; specifically targetting low-medium contributor activity, medium-hgih usage of the project, with only a handful of maintainers
+
+<!-- Previous points
 - Themed from MMORPG gaming, gamifiying Mob programming OSS repos
 - Look for criteria in OSS repos 
   1. Low to medium contributor activity, only a handful of maintainers
   2. Medium to high usage of the OSS 
   3. Our impact can be potentially high; in a few days
+-->
+
+## Join Us! :smiling_face_with_three_hearts:
+
+Anoyone is welcome to participate in our Raids - no matter your experience level! Never contributed to an OSS project? No worries! We are all learning something or another, so come along for the ride!
+
+### Intake Quests
+
+1. Say hello to your fellow raiders in the #:bee:-raid-general channel on [Discord][discord]
+2. Raise your first issue against a raid, and start contributing!
+
+### Advancements
+
+When you are:
+* Actively contributing to Raids
+* Being an active community member on the KCD Discord Server
+
+Then we can determine responsibilities and get you running as either an Org member or Collab (reduced permissions)
+
+## Raids :crossed_swords:
+
+* One Raid at a time to maximize impact in a short amount of time.
+
+### Communication
+
+We primarily communicate via [Discord][discord] in the `OSS-RAID-GROUP` category
+
+### Workflow
 
-## Joining KCD Raid Group :smiling_face_with_three_hearts:
-1. Active Raid contributor
-2. Active Community member (KCD) 
-3. Determine responsibilities then set to either Org member or Collab (reduced permissions)
+When participating in a Raid, use the following steps.
 
-## The Process find target -> communicate -> Raid -> Push Code :fire:
-1. One Target at a time to maximize impact in short amount of time.
+1. Raise an issue in the Raid repo with a one-liner title describing what you are working on - for example `Migrate page-header/breadcrumbs tests to testing-library`
+2. Fork the Raid Repo
+3. Create a branch named after your GitHub username - e.g. `nobrayner`
+4. Hack away :sunglasses::computer:
+5. Create a PR titled the same as your issue, and a desciption of `Fixes #{ISSUE_NO}` - [like this PR for example](https://github.com/kcdraidgroup/hospitalrun-frontend/pull/8)
 
-## Communication :smiley:
-1. Primarily www.kentcdodds.com/discord #oss-raid-group channel 
-2. Projects tab per Repo we are working on, to kanban who is working on what to prevent redudant work or stepping on toes.
 
-## Workflow :sunglasses:
+[discord]: https://kentcdodds.com/discord/
```

Comment:
You can create links to specific channels in discord by using the "Create Invite" feature.  

![image](https://user-images.githubusercontent.com/23029903/102343725-fdef0b00-3fee-11eb-8552-1ad3f8c8578c.png)

Just a thought so new-comers don't get too lost on the big server (they might be new to discord too).

Reply:
Kent's Discord Bot I think prevents that, but we can try.

Reply:
I was debating doing a direct link... We can give it a go 👍

Reply:
Done